README.md

```diff
@@ -426,6 +444,23 @@ branch.
 - `base_url` (default: `https://api.github.com`) is the base URL to the Github
   API to use (for Github Enterprise support set this to `https://{your_domain}/api/v3`).
 
+### `all_repos.push.bitbucket_server_pull_request`
+
+Pushes the branch to `origin` and then creates a Bitbucket pull request for the branch.
+
+#### Required `push_settings`
+
+- `base_url`: the Bitbucket server URL (eg `bitbucket.domain.com`)
+- `username`: the Bitbucket username you will log in as.
+- `app_password`: the authentication method for the above user to login with
+    - Create an application password within your [account settings](https://bitbucket.domain.com/plugins/servlet/access-tokens/manage).
+    - We need the scope: Repositories -> Read
```

Comment:
I'm guessing this actually needs Write to make a pull request? or is Read actually ok here?

Reply:
`REPO_READ` is enough for creating a PR according to the documentation and my tests (see https://docs.atlassian.com/bitbucket-server/rest/7.11.2/bitbucket-rest.html#idp291)